spec/services/parse/bing_parse_service_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Parse::BingParseService, type: :service do
+  describe '#parse' do
+    context 'given the html_code is valid' do
+      it 'returns parse result', vcr: 'services/search/bing/valid' do
+        search_service = Search::BingSearchService.new(keyword: 'watch')
+        html_code = search_service.search
+
+        parse_service = described_class.new(html_code)
+        parse_result = parse_service.parse
+
+        expect(parse_result).to eq(
+          {
+            adwords_top_urls: [
```

Comment:
This g=hurts the test readability. Can you edit the VCR recording to simplify this data?

Reply:
@olivierobert I could be wrong, but it's not possible to edit the VCR recording, because the body is encoded:

<img width="1052" alt="Screen Shot 2023-06-23 at 10 52 01" src="https://github.com/nimblehq/ic-rails-huey-toby/assets/18277915/31359c3a-3e43-4f35-9db9-6bd7ac33af03">

However, I can re-fator the URLS in the a constant, what do you think? 🙏

Reply:
Or... I guess I could try to decode it, edit it, and encode it again 💭